app/config/message.js

```diff
@@ -0,0 +1,67 @@
+import convict from 'convict'
+import environments from '../constants/environments.js'
+
+const config = convict({
+  messageQueue: {
+    host: {
+      doc: 'Message queue host URL',
+      format: 'url',
+      default: null,
+      env: 'MESSAGE_HOST'
+    },
+    username: {
+      doc: 'Message queue username',
+      format: String,
+      default: null,
+      env: 'MESSAGE_USER'
+    },
+    password: {
+      doc: 'Message queue password',
+      format: String,
+      default: null,
+      env: 'MESSAGE_PASSWORD',
+      sensitive: true
+    },
+    useCredentialChain: {
+      doc: 'Use credential chain for authentication',
+      format: Boolean,
+      default: false,
+      env: 'USE_CREDENTIAL_CHAIN'
+    },
+    managedIdentityClientId: {
+      doc: 'Managed identity client ID',
+      format: String,
+      default: null,
+      env: 'COMMS_AZURE_CLIENT_ID',
```

Comment:
Do we need COMMS_ prefix?  Just using the default one will simplify some use of managed identities as packages will auto identify it

Reply:
Ah yeah - my reasoning for this is that when we were on ADP, we didn't need this value in our `.env` or `appConfig` etc. as they handled that particular variable, but now that we're on FCP I had to take the Client ID value from the Azure portal and realised if we need the Client ID for other services we'd need to distinguish between them within our `.env` files?

Reply:
I don't think we'd need to reference a client id for another microservice in a repo.  However, we may need a client id for things like APIM or other integrations.  I'd say for those, then we come up with a prefix such as APIM_CLIENT_ID to allow us to use the default for our more common use case.

Reply:
Understood, have updated the variable name: `COMMS_AZURE_CLIENT_ID` -> `AZURE_CLIENT_ID`.